Tests/XCUITests/PerformanceTests.swift

```diff
@@ -169,6 +175,70 @@ class PerformanceTests: BaseTestCase {
         }
     }
 
+    func testPerfBookmarks1startUp() {
```

Comment:
Do we also want one test like this for History?

Reply:
I see, it is in a different ticket and PR, sorry!

Reply:
no worries! i'm testing the new suite of tests now. just have to figure out the best way to get good performance numbers after populating the entire 1k and 4k cells in `History Panel`